### ISSUE TITLE
unloading jobs does not stop them

### ISSUE THREAD

**Issue by mheily** [mheily/jobd#49]
jobs aren't actually killed when they are unloaded, even though SIGTERM is sent.

```
 launchd[3347]: job_unload(job.c:451): sending SIGTERM to process group 3371
aunchd[3347]: manager_unload_job(manager.c:286): job kmix unloaded

% ps ax|grep 3371
3371  -  Ss    0:02.21 /usr/local/bin/kmix --nofork
3818  3  S+    0:00.00 grep 3371
```

**Comment by mheily** [comment]
it turns out kmix refuses to die if you send it SIGTERM.

**Comment by mheily** [comment]
all jobs now are immune to kills. I think they are inheriting the signal handling of the parent, which is to SIG_IGN.

**Comment by mheily** [comment]
fixed as of 296aeb0
